index.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
     "prefer-destructuring": ["warn", {"object": true, "array": true}],
     "arrow-parens": ["error", "as-needed", { "requireForBlockBody": false }],
     "radix": ["error", "as-needed"],
-    "comma-dangle": ["error", "only-multiline"]
+    "comma-dangle": ["error", "always-multiline"]
```

Comment:
tem que verificar se isso não vai quebrar no monolito, agora lembrei @fsblemos porque a gente tirou, se for mergear aqui lá no monólito vc vai ter que desativar, eu acho!

Reply:
que acha de colocar só no ca-components então?

Reply:
Eu acho que deveria ser padrão sempre ter a vírgula e, no `.eslintrc` do ContaAzul, desabilitamos por hora.

Reply:
se pá nem vai quebrar, pois já tem uma regra lá no eslint do monolito:

```json
"comma-dangle": ["error", "never"],
```